src/shared/api/http.ts

```diff
@@ -11,58 +24,46 @@ async function get<T>(path: string, params?: PaginationParams): Promise<T> {
   return res.json();
 }
 
-export function fetchAccounts(params?: PaginationParams): Promise<Paginated<Account>> {
+export function fetchAccounts(params?: PaginationParams): Promise<Paginated<AccountDto>> {
   return get('/accounts', params);
```

Comment:
Use your schemas to validate and transform data from backend!

```suggestion
  return get('/accounts', params).then((raw) => accountsSchema.parse(raw));
```